lib/instrumentation/index.js

```diff
@@ -68,7 +68,7 @@ function Instrumentation (agent) {
   this._patches = new NamedArray()
 
   for (let mod of MODULES) {
-    this.addPatch(mod, require.resolve(`./modules/${mod}`))
+    this.addPatch(mod, require(`./modules/${mod}`))
```

Comment:
```suggestion
    this.addPatch(mod, (exports, name, version, enabled) => {
      // Lazy require so that we don't have to use `require.resolve` which
      // would fail in combination with Webpack. For more info see:
      // https://github.com/elastic/apm-agent-nodejs/pull/957
      const patch = require(`./modules/${mod}`)
      return patch(exports, name, version, enabled)
    })
```